vllm/entrypoints/openai/protocol.py

```diff
@@ -476,6 +498,11 @@ def to_sampling_params(
                 json_schema = self.response_format.json_schema
                 assert json_schema is not None
                 self.guided_json = json_schema.json_schema
+            elif self.response_format.type == "structural_tag":
+                structural_tag = self.response_format
+                assert structural_tag is not None
```

Comment:
Also please assert that isinstance of StructuralTagResponseFormat

Reply:
done!